### PR TITLE
docs: LDAP authn/authz

### PR DESCRIPTION
**Pull request by lafirest**

### CODE REVIEW

**Review by wivwiv — APPROVED** [review]

File: dir.yaml [hunk 1]
```diff
@@ -472,6 +473,7 @@
             - access-control/authz/postgresql
             - access-control/authz/redis
             - access-control/authz/mongodb
+            - access-control/authz/ldap
```
Comment (wivwiv) [inline dir.yaml]:
```suggestion
```

File: dir.yaml [hunk 2]
```diff
@@ -457,6 +457,7 @@
                 - access-control/authn/postgresql
                 - access-control/authn/redis
                 - access-control/authn/mongodb
+                - access-control/authn/ldap
```
Comment (wivwiv) [inline dir.yaml]:
```suggestion
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

**Review by wivwiv — APPROVED** [review]

**Comment by wivwiv** [comment]
@Meggielqk Please commit this section to release-5.2.

**Comment by Meggielqk** [comment]
> @Meggielqk Please commit this section to release-5.2.

See https://github.com/emqx/emqx-docs/pull/2083

**Pull request closed without merging**